dc-model/src/main/java/de/digitalcollections/model/list/filtering/FilterLogicalOperator.java

```diff
@@ -0,0 +1,25 @@
+package de.digitalcollections.model.list.filtering;
+
+import java.util.stream.Stream;
+
+public enum FilterLogicalOperator {
+  AND("$AND"),
+  OR("$OR");
+
+  private String operand;
+
+  public static FilterLogicalOperator fromUrlOperand(String value) {
```

Comment:
Das würde ich eher außerhalb des Modells sehen, weil hier ein spezifischer Anwendungsfall (URL-Syntax) behandelt wird, der aber eigentlich erst im Service selber zum Tragen kommt, und sollte eine Anwendung sich für eine andere URL-Syntax entscheiden, paßt das nicht mehr mit dem Modell zusammen.

Reply:
Sowas haben wir in der `FilterOperation` auch so :thinking: Müsste die dann nicht auch überarbeitet werden?

Reply:
Ich hab's umbenannt und das '$' entfernt.